tests/unit/Codeception/Lib/ParserTest.php

```diff
@@ -146,13 +146,80 @@ public function testParseFileWhichUnsetsFileVariable()
         $this->assertSame([], $classes);
     }
 
+    #[Group('core')]
+    public function testParseExceptionWithFileNameOnly()
+    {
+        $this->expectException(\Codeception\Exception\TestParseException::class);
+        $this->expectExceptionMessage("Couldn't parse test 'test.file'");
+        throw new \Codeception\Exception\TestParseException('test.file');
+    }
+
+    #[Group('core')]
+    public function testParseExceptionWithErrors()
+    {
+        $this->expectException(\Codeception\Exception\TestParseException::class);
+        $this->expectExceptionMessage("Couldn't parse test 'test.file':\nFunny error");
```

Comment:
Your tests fail because there is no `:` in error exception message.

Reply:
@Naktibalda thank you so much. Yes, when I was adapting to the latest 5.0 branch, I forgot to re-add the colon there.

I've now fixed that and also used `PHP_EOL` rather than `\n` in the test, so that they should not fail on other OS types.